Common/src/config_structure.cpp

```diff
@@ -2256,26 +2206,37 @@ void CConfig::SetConfig_Options() {
   /* DESCRIPTION: Determines if the multizone problem is solved for time-domain. */
   addBoolOption("TIME_DOMAIN", Time_Domain, false);
   /* DESCRIPTION: Number of outer iterations in the multizone problem. */
-  addUnsignedLongOption("OUTER_ITER", Outer_Iter, 1);
+  addUnsignedLongOption("OUTER_ITER", nOuterIter, 1);
   /* DESCRIPTION: Number of inner iterations in each multizone block. */
-  addUnsignedLongOption("INNER_ITER", Inner_Iter, 1);
+  addUnsignedLongOption("INNER_ITER", nInnerIter, 1);
+  /* DESCRIPTION: Number of adjoint inner iterations in each multizone block. */
+  addUnsignedLongOption("INNER_ITER_ADJOINT", nInnerIter_Adjoint, 1);
```

Comment:
Can you re-use the "normal" INNER_ITER?

Reply:
Yes. The advantage would be that it's more elegant in the code. This disadvantage being that one cannot elegantly change `DIRECT` to `DISCRETE_ADJOINT` to compute adjoints but that one has to go though all config files and adapt `INNER_ITER`. 
To me it's a draw. No problem if you still would like to have it removed.

Reply:
That is what I would like to avoid, having to change more than one option when switching from DIRECT to ADJOINT, I think others would agree since there has been so much work in making Outer_Iter, Time_Iter, etc. consistent between single and multizone.

Reply:
Ok. I'll change that.
But let's just think what can be done to avoid touching all config files (think, e.g., of a case with not two but quite a few zones) whenever ones switches from `DIRECT` to `DISCRETE_ADJOINT`.

Reply:
Old up, maybe I got something wrong as it feels we are saying the same thing with opposite meanings.
For a direct multizone problem INNER_ITER is how many (e.g.) fluid iterations are performed per coupling iteration, right?
What is the meaning of INNER_ITER_ADJOINT?

Reply:
Right. Same for the adjoint, that is how many discrete adjoint (flow) solution iterations are performed before moving on to the next zone (i.e. the diagonal part, given the coupling contributions from a previous iteration).
They don't have to be necessarily the same, I'm lacking some experience here. (CHT problems are very stable in that regard.)

Reply:
Ok that's what I thought, so the testcases that would be problematic to change are your own existing ones?
I am thinking that from the user's perspective it would be annoying to have two sets of options in each file for the "same" thing (we have it for the linear solver though...)
We can wait for other opinions.

Reply:
My test cases are not problematic in the sense that I'm playing around with a bunch of different options anyway. Let's just go with `INNER_ITER` as we have at least one preference. Changes can be made quickly.

Reply:
I agree with @pcarruscag here, given that we have INNER_ITER, I don't see a reason for duplicating the option for the adjoint. Specially since the direct iterations to be recorded are always 1 on the adjoint solution.